pcdet/models/detectors/detector3d_template.py

```diff
@@ -364,14 +364,16 @@ def load_params_from_file(self, filename, logger, to_cpu=False, pre_trained_path
 
         logger.info('==> Loading parameters from checkpoint %s to %s' % (filename, 'CPU' if to_cpu else 'GPU'))
         loc_type = torch.device('cpu') if to_cpu else None
-        checkpoint = torch.load(filename, map_location=loc_type)
-        model_state_disk = checkpoint['model_state']
+        # checkpoint = torch.load(filename, map_location=loc_type)
```

Comment:
Note: the published model weights shared for VoxelNext did _not_ follow the pattern this code expected, and instead the shared file includes *only* the model state dict.

The hack made here was required in order to load their pre-published weights.